lib/ansible/modules/storage/netapp/na_ontap_node.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/python
+
+# (c) 2018, NetApp, Inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+module: na_ontap_node
+short_description: NetApp ONTAP Rename a node.
+extends_documentation_fragment:
+    - netapp.na_ontap
+version_added: '2.7'
+author: NetApp Ansible Team (ng-ansibleteam@netapp.com)
+description:
+- Rename an ONTAP node.
+options:
+  name:
+    description:
+    - The new name for the node
+    required: true
+
+  from_name:
+    description:
+    - The name of the node to be renamed.
+    required: true
+
+'''
+
+EXAMPLES = """
+- name: rename node
+  na_ontap_node:
+    hostname: "{{ netapp_hostname }}"
+    username: "{{ netapp_username }}"
+    password: "{{ netapp_password }}"
+    from_name: laurentn-vsim1
```

Comment:
What happens if this playbook is run twice, ie if `laurentn-vsim1` doesn't exist on the 2nd run?

Code needs to deal wit that and the documentation needs to explain it